### PR TITLE
Rename system status app

### PR DESCRIPTION
**Pull request by ashton22305**
Renames the system status app to prevent conflicts with ondemand-systemstatus

### CODE REVIEW

**Comment by treydock** [comment]
Looking at 3939de2752c14a2a566c4d9c186f49720ad17af9 , does the route path need adjustments?  If so then also the manifest's URL?  Otherwise these may collide with existing app I'd imagine.

**Comment by ashton22305** [comment]
I don't think they'd collide because the existing app uses the route `/apps/show/systemstatus`, but it would probably be a good idea anyway for consistency.

File: apps/dashboard/config/routes.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@
     delete '/activejobs' => 'active_jobs#delete_job', :as => 'delete_job'
   end
 
-  get '/systemstatus', to: 'system_status#index', as: 'system_status' if Configuration.can_access_systemstatus?
+  get '/system-status', to: 'system_status#index', as: 'system_status' if Configuration.can_access_systemstatus?
```
Comment (treydock) [inline apps/dashboard/config/routes.rb:101]:
```suggestion
  get '/system-status', to: 'system_status#index', as: 'system_status' if Configuration.can_access_system_status?
```

File: apps/dashboard/config/configuration_singleton.rb [hunk 2]
```diff
@@ -321,7 +321,7 @@ def can_access_projects?
   end
 
   def can_access_systemstatus?
```
Comment (treydock) [inline apps/dashboard/config/configuration_singleton.rb:323]:
```suggestion
  def can_access_system_status?
```

**Pull request merged**